src/app/components/FormatCredit/index.lazy.scss

```diff
@@ -1,23 +1,52 @@
 @import '../../constants';
 
 .FormatCredit {
-  color: $color-black-transparent-75;
-  font-family: var(--od-font-stack-sans);
+  color: $color-black;
```

Comment:
I'd suggest using the `--od` var for this, then you shouldn't need the `invert` override below

Reply:
Thanks, fixed.